plugins/CoreHome/templates/_dataTableActions.twig

```diff
@@ -103,6 +103,20 @@
             </a>
         {% endif %}
 
+        {% if properties.datatable_actions|default is not empty %}
+        {% for action in properties.datatable_actions %}
+            <a class='dataTableAction {{ action.id|e('html_attr') }}'
+               href='javascript:;' title="{{ action.title|e('html_attr') }}"
+            >
+                {% if action.icon starts with 'icon-' %}
+                    <span title="" class="{{ action.icon }}"></span>
```

Comment:
any reason for not adding the title here? That prevents the tooltip from being displayed when hovering the icon directly.

Reply:
No, probably a typo